dataactvalidator/config/sqlrules/b11_object_class_program_activity_3.sql

```diff
@@ -3,4 +3,5 @@ SELECT
     op.object_class
 FROM object_class_program_activity AS op
 WHERE op.submission_id = {}
-AND op.object_class = '000'
+AND LENGTH(op.object_class) <> 3
+AND op.object_class <> '0'
```

Comment:
It should never be "0" because of our padding. We can also probably just change this back since we still need it to be a warning.